BSPConversionLib/Source/EntityConverter.cs

```diff
@@ -551,6 +551,50 @@ private void GiveWeaponOnStartTouch(Entity trigger, Entity weaponEnt)
 				fireOnce = -1
 			};
 			trigger.connections.Add(connection);
+
+			if (weaponEnt["count"] == "-1")
```

Comment:
Extract lines 555 - 576 into another method called `GiveWeaponAmmoOnStartTouch` since this method is getting too big.